### DISCUSSION TITLE
How can I disable this warning screen?

### DISCUSSION THREAD

**Discussion by gnuchanos** [qutebrowser/qutebrowser#7690] [Q&A]
![2023-05-05-175911_1280x1024_scrot](https://user-images.githubusercontent.com/117280480/236494834-4f29a059-b7c5-4c7f-a70e-26bbff22b514.png)
How can I disable this warning screen? It doesn't close even if the internet goes out for just 10 seconds, it keeps popping up repeatedly.

**Comment by toofar** [comment]
Try `:set content.tls.certificate_errors ask-block-thirdparty` and see if that helps. (It says to try that in the error message too btw!)
Note that that setting does support URL patterns (so you can do, for example, `:set -u www.youtube.com content.tls.certificate_errors block` if it turns out to be failing top level navigations).
